codegen/test_data/clients/bar.gogen

```diff
@@ -31,11 +31,10 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
-	zanzibar "github.com/uber/zanzibar/runtime"
```

Comment:
gogen lost its purpose when we moved ochestrator out zanzibar, I would actually prefer to have unit tests / integration tests verify I/O instead of the content of generated go code

Reply:
Good point, that's on the todo list.